### DISCUSSION TITLE
Persist changes to a font

### DISCUSSION THREAD

**Discussion by tiagohermano** [fontra/fontra#126] [Q&A] [answered]
Is there a way to persist the changes made to a font?

**Comment by justvanrossum** [comment, marked as answer]
It saves changes to UFO files (immediately), but not in TTF or OTF.

**Reply by navv-1** [reply]
Hi @justvanrossum, does fontra also prevent edit ttf/otf font? I got this message: 
![image](https://github.com/googlefonts/fontra/assets/80741557/d3f34397-cbda-44fb-beca-bfa0fad91678)

**Reply by justvanrossum** [reply]
Yes, because it is not able to save changes (it can't write TTF). In the future, we will provide a conversion option, so you can save it in a format that Fontra _can_ edit.
